src/toggle/toggle.tsx

```diff
@@ -1,36 +1,65 @@
+import { ChevronDownIcon } from "@lifesg/react-icons/chevron-down";
+import { ChevronUpIcon } from "@lifesg/react-icons/chevron-up";
 import { useEffect, useRef, useState } from "react";
+import { ToggleIcon, ToggleIconType } from "../shared/toggle-icon/toggle-icon";
+import { TextList } from "../text-list";
+import { SimpleIdGenerator } from "../util";
 import {
+    AlertContainer,
+    Children,
+    ChildrenContainer,
     Container,
+    ErrorListContainer,
+    ErrorListItem,
+    ErrorListli,
+    HeaderContainer,
+    IndicatorLabelContainer,
     Input,
     Label,
+    RemoveButton,
     SubLabel,
     TextContainer,
+    ViewMoreOrLessButtonContainer,
+    ViewMoreOrLessButtonLabel,
 } from "./toggle.styles";
 import { ToggleProps } from "./types";
-import { ToggleIcon, ToggleIconType } from "../shared/toggle-icon/toggle-icon";
-import { SimpleIdGenerator } from "../util";
 
 export const Toggle = ({
     type = "checkbox",
     indicator,
     checked,
     styleType = "default",
     children,
+    childrenMaxLines,
     subLabel,
     disabled,
     error,
     name,
     id,
     className,
+    compositeSection,
+    removable,
+    onRemove,
     "data-testid": testId,
     onChange,
 }: ToggleProps) => {
     // =============================================================================
     // CONST, STATE, REF
     // =============================================================================
-    const [selected, setSelected] = useState<boolean>(checked);
+    const {
+        collapsible = true,
+        errors,
+        children: compositeOptionSection,
+        show: showCompositeOptionSection,
```

Comment:
keeping the naming consistent
```suggestion
        children: compositeSectionChildren,
        show: showCompositeSection,
```